docsite/rst/playbooks_best_practices.rst

```diff
@@ -27,8 +27,8 @@ Directory Layout
 
 The top level of the directory would contain files and directories like so::
 
-    production                # inventory file for production servers
-    staging                   # inventory file for staging environment 
+    production                # inventory file for production servers (or directory that contains hosts.yml)
+    staging                   # inventory file for staging environment (or directory that contains hosts.yml)
```

Comment:
if it is a directory it won't pick up group_vars, we use a 2nd example with directories to reflect this